.goreleaser.yml

```diff
@@ -17,6 +17,8 @@ builds:
       - arm64
     main: .
     goamd64:
+      - v1
```

Comment:
why we need to specify all of them ? when no leave it  as default ? I don't think we really need all of the variants like that, we need one that works on all of them.

Reply:
There is a 5-10% improvement when building with optimizations, and when there is a docker image thats supported by the arch, that one will be pulled, we dont lose anything when building all of them